central/role/resources/list.go

```diff
@@ -66,7 +66,6 @@ var (
 	AllComments = newDeprecatedResourceMetadata("AllComments", permissions.GlobalScope,
 		Administration)
 	APIToken     = newDeprecatedResourceMetadata("APIToken", permissions.GlobalScope, Integration)
-	AuthPlugin   = newDeprecatedResourceMetadata("AuthPlugin", permissions.GlobalScope, Access)
```

Comment:
There still are references to AuthPlugin in `qa-tests-backend` (1) and in `ui` (4 different files, not counting indirect references) that can be cleaned as well.

Reply:
Yup, removed.

Reply:
I think we should have a separated PR for UI and QA tests. Removing backend code should be a final PR to clean everything up.